Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -909,14 +909,15 @@ wrapParseWarning fp pw = PackageDistSuspicious (ParseWarning fp pw)
 -- each of those branch will be checked one by one.
 extractAssocDeps
   :: UnqualComponentName -- Name of the target library
-  -> CondTree ConfVar [Dependency] Library
+  -> CondTree ConfVar Dependencies Library
   -> AssocDep
 extractAssocDeps n ct =
   let a = ignoreConditions ct
    in -- Merging is fine here, remember the specific
       -- library dependencies will be checked branch
       -- by branch.
-      (n, snd a)
+      -- MP: TOOD: WRONG
```

Comment:
What is the plan about what to do here?

Reply:
This function does:

> Gets a list of dependencies from a Library target to pass to PVP related functions.

Should PVP care about private dependencies of a library? For API reasons it shouldn't matter at all, however, unfortunately, it may matter for semantic reasons, since a PVP breaking behaviour change in a private dependency may impact the PVP of the parent package?